src/param.rs

```diff
@@ -5,15 +5,21 @@
 //! Supports optional parameters for calling SQS and ETS.
 
 use std::collections::BTreeMap;
-pub type Params = BTreeMap<String, String>;
+pub type Params = BTreeMap<String, Option<String>>;
```

Comment:
This is necessary because S3 has some URLs in the format `/foo?bar` where the key in the query string has no associated value.